### PR TITLE
JBEAP-24957 Removing Reset test for Configuration => Subsystem => Distributable Web => Configuration

### PR DESCRIPTION
**Pull request by mchoma**
See https://issues.redhat.com/browse/JBEAP-24957 for details.

### CODE REVIEW

**Comment by mchoma** [comment]
@spriadka please review. I am not able to assign for some reason.

File: packages/testsuite/cypress/e2e/distributable-web/test-configuration-subsystem-distributable-web-configuration.cy.ts [hunk 1]
```diff
@@ -126,12 +126,6 @@ describe("TESTS: Configuration => Subsystem => Distributable Web => Configuratio
     );
   });
 
-  it("Reset", () => {
-    cy.navigateTo(managementEndpoint, "distributable-web");
-    cy.get("#dw-configuration-item").click();
-    cy.resetForm(configurationFormId, managementEndpoint, ["subsystem", "distributable-web"]);
```
Comment (mchoma) [inline packages/testsuite/cypress/e2e/distributable-web/test-configuration-subsystem-distributable-web-configuration.cy.ts:132]: Alternatively this line can be written to expect current warning, but don't know how to write that.

**Comment by OndrejKotek** [comment]
@mchoma Can you update the description of the PR and commit to be more descriptive please? Something like _"JBEAP-24957 Removing Reset test for Configuration => Subsystem => Distributable Web => Configuration"_ could work.

**Comment by mchoma** [comment]
> Can you update the description of the PR and commit

Done.

File: packages/testsuite/cypress/e2e/distributable-web/test-configuration-subsystem-distributable-web-configuration.cy.ts (hunk 1 above)
Comment (OndrejKotek) [reply packages/testsuite/cypress/e2e/distributable-web/test-configuration-subsystem-distributable-web-configuration.cy.ts:132]: Let's remove the test for now. We'll update this method in scope of https://github.com/hal/berg/issues/44.

**Pull request merged**